keyboards/dz60/keymaps/olivierko/keymap.c

```diff
@@ -0,0 +1,87 @@
+#include QMK_KEYBOARD_H
+
+enum keyboard_layers {
+  _BL = 0,
+  _SL = 1,
+  _FL = 2,
+  _CL = 3,
+};
+
+enum custom_keycodes {
+  SWE_AA = SAFE_RANGE,
+  SWE_AE,
+  SWE_OE,
+};
+
+char *alt_codes[][2] = {
+    {
+        SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_2)SS_TAP(X_KP_2)SS_TAP(X_KP_9)), // Alt+0229 → å
+        SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_1)SS_TAP(X_KP_9)SS_TAP(X_KP_7)), // Alt+0197 → Å
+    },
+    {
+		SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_2)SS_TAP(X_KP_2)SS_TAP(X_KP_8)), // Alt+0228 → ä
+        SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_1)SS_TAP(X_KP_9)SS_TAP(X_KP_6)), // Alt+0196 → Ä
+    },
+    {
+        SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_2)SS_TAP(X_KP_4)SS_TAP(X_KP_6)), // Alt+0246 → ö
+        SS_LALT(SS_TAP(X_KP_0)SS_TAP(X_KP_2)SS_TAP(X_KP_1)SS_TAP(X_KP_4)), // Alt+0214 → Ö
+    },
+};
```

Comment:
This is a very clever implementation, and I may steal this for my own keymap in future. :grinning:

Reply:
Haha, unfortunately I can't take credit for it as I stumbled across it on reddit I think.